hieradata/common.yaml

```diff
@@ -458,6 +458,11 @@ govuk::apps::rummager::rabbitmq::enable_publishing_listener: true
 govuk::apps::rummager::redis_host: 'api-redis-1.api'
 govuk::apps::rummager::redis_port: '6379'
 
+govuk::apps::search_admin::db_name: 'search_admin_production'
+govuk::apps::search_admin::db_hostname: 'master.mysql'
+govuk::apps::search_admin::db_password: "%{hiera('govuk::apps::search_admin::db::password')}"
```

Comment:
Yes good catch. This is actually coming from the encrypted hieradata with the same name so we don't need to have it here as well.